hyprhook/src/main.cpp

```diff
@@ -5,6 +5,8 @@
 
 #include <hyprland/src/plugins/PluginAPI.hpp>
 #define private public
+#include <hyprland/src/helpers/Color.hpp>
+#include <hyprland/src/helpers/Timer.hpp>
```

Comment:
What's the reason behind timing being included or is this remanence code from trying stuff out?